content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,409 @@
+decreasing at a certain checkpoint and you want to make some changes to fix
+that.
+
+First, we need to apply the checkpoint we want to begin our new experiment from.
+To do that, run the following command:
+
+```bash
+dvc exp apply b3de55f
+```
+
+where _b3de55f_ is the id of the checkpoint you want to reference.
+
+Next, we'll change the learning rate in the _params.yaml_ to `0.001`. Then we'll
+start a new experiment based on an existing checkpoint with the following
+command:
+
+```bash
+dvc exp run
+```
+
+You'll be able to see where the experiment starts from the existing checkpoint
+by running:
+
+```bash
+dvc exp show
+```
+
+You should seem something similar to this in your terminal.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃ step ┃     loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -            │   18 │ 0.12399  │ 0.9625 │ 473987 │ 0.001  │ 0            │
+│ main                  │ Apr 27, 2021 │    - │       -  │      - │ 473987 │ 0.0001 │ 0            │
+│ │ ╓ exp-63287         │ 04:37 PM     │   18 │ 0.12399  │ 0.9625 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 868c144           │ 04:37 PM     │   17 │ 0.12042  │ 0.9616 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 4de4e69           │ 04:37 PM     │   16 │ 0.13808  │ 0.9585 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ d8d633c           │ 04:37 PM     │   15 │ 0.20415  │ 0.9397 │ 473987 │ 0.001  │ 0            │
+│ │ ╟ 8fbab93 (b3de55f) │ 04:36 PM     │   14 │ 0.24509  │ 0.9309 │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-4de2a         │ 05:03 PM     │    7 │  0.3096  │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ d50c724           │ 05:03 PM     │    6 │ 0.33246  │ 0.9044 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 29491a9           │ 05:02 PM     │    5 │ 0.36601  │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b3de55f           │ 05:02 PM     │    4 │ 0.41666  │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+```
+
+The existing checkpoint is referenced at the beginning of the new experiment.
+The new experiment is referred to as a modified experiment because it's taking
+existing data and using it as the starting point.
+
+### Metrics diff
+
+When you've run all the experiments you want to and you are ready to compare
+metrics between checkpoints, you can run the command:
+
+```bash
+dvc metrics diff 8fbab93 868c144
+```
+
+Make sure that you replace `8fbab93` and `868c144` with checkpoint ids from your
+table with the checkpoints you want to compare. You'll see something similart to
+this in your terminal.
+
+```dvc
+Path          Metric    Old      New      Change
+dvclive.json  acc       0.9854   0.9831   -0.0023
+dvclive.json  loss      0.04828  0.05095  0.00267
+dvclive.json  step      19       15       -4
+```
```

Comment:
You can reference that these are the same numbers as you see in the table but in a different format.